portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,8 +42,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String text = getParameter(request, "textInput", "default2");
     hardcoded.add(text);
     String json= new Gson().toJson(hardcoded);
-    response.setContentType("application/json;");
 
+    long timestamp = System.currentTimeMillis();
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("title", "Comments");
+    taskEntity.setProperty("Comments:", text);
+    taskEntity.setProperty("timestamp", timestamp);
```

Comment:
Rather than calling the Entity a "Task" and setting its title to "Comments" you might consider just using an entity called "Comments". E.g.
```suggestion
    Entity entity = new Entity("Comments");
    entity.setProperty("comment", text);
    entity.setProperty("timestamp", timestamp);
```

Reply:
fixed in later commit

Reply:
Fixed in later commit